cmd/ci-secret-bootstrap/main.go

```diff
@@ -233,14 +233,18 @@ func constructSecrets(config []secretConfig, bwClient bitwarden.Client) (map[str
 			data[key] = value
 		}
 		for _, secretContext := range secretConfig.To {
+			t := secretContext.Type
+			if t == "" {
+				t = coreapi.SecretTypeOpaque
```

Comment:
Why not changing the `secretContext.Type` directly? Is it going to be used in somewhere else?

Reply:
Updated.